<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="rohith" ignored="0" name="Default test" tests="8" failures="0" timestamp="21 Feb 2019 03:01:34 GMT" time="20.13" errors="0">
  <testcase name="@AfterTest closeHomePage" time="0.001" classname="project.circuit.automation.HomePageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at project.circuit.automation.HomePageTest.closeHomePage(HomePageTest.java:63)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.afterRun(TestRunner.java:725)
at org.testng.TestRunner.run(TestRunner.java:509)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterTest closeHomePage -->
  <testcase name="signInTest" time="3.158" classname="project.circuit.automation.HomePageTest"/>
  <testcase name="loginTest" time="0.272" classname="project.circuit.automation.HomePageTest"/>
  <testcase name="passwordTest" time="2.411" classname="project.circuit.automation.HomePageTest"/>
  <testcase name="conversation" time="1.99" classname="project.circuit.automation.HomePageTest"/>
  <testcase name="createFrom" time="0.118" classname="project.circuit.automation.HomePageTest"/>
  <testcase name="enterTitle" time="0.4" classname="project.circuit.automation.HomePageTest"/>
  <testcase name="enterDescription" time="0.156" classname="project.circuit.automation.HomePageTest"/>
  <testcase name="startButtontest" time="0.076" classname="project.circuit.automation.HomePageTest"/>
</testsuite> <!-- Default test -->
